docs/user-manual/framework/memory-management/buffer-pool.md

```diff
@@ -1,15 +1,15 @@
-# Dynamic Memory and Buffer Management
+# Buffer Pools with Svc.BufferManager
 
 In embedded systems, dynamic memory allocation (a.k.a heap allocation) is typically avoided to reduce the steady-state
 variability in a running system. Avoiding dynamic memory allocation also avoids the problem of what to do in the case of
 a failed allocation. However, sometimes dynamic allocation provides for a simpler or more efficient solution.
```

Comment:
However, sometimes dynamic allocation is required for a more robust or efficient solution.